modules/mono/editor/GodotTools/GodotTools/Internals/Internal.cs

```diff
@@ -59,6 +66,12 @@ public static bool IsMacOSAppBundleInstalled(string bundleId)
         public static void EditorPlugin_AddControlToEditorRunBar(Control control) =>
             godot_icall_Internal_EditorPlugin_AddControlToEditorRunBar(control.NativeInstance);
 
+        public static void EditorPlugin_PopupToast(string message, PopupSeverity severity = PopupSeverity.SEVERITY_INFO)
```

Comment:
- Usually the name of these methods is `{TypeName}_{MethodName}`, although in this case we don't have an `EditorToaster` type. I still think we should try to avoid deviating too much from the name in Core, otherwise it makes it more difficult in the future to understand where this method is coming from.
- To match the method in Core, also add the `p_tooltip` parameter.

```suggestion
        public static void EditorToaster_PopupStr(string message, EditorToasterSeverity severity = EditorToasterSeverity.Info, string? tooltip = null)
```